web/templates/pages/tasks.html

```diff
@@ -0,0 +1,48 @@
+{% extends 'base.html' %}
+
+{% block title %}All CodeCatz Tasks{% endblock %}
+
+{% block content%}
+
+from django import forms
+from .models import Task
```

Comment:
@ialja i think its a mistake from the next thing i'm working on ... deleted it :)